ports/openvino/portfile.cmake

```
@@ -1,3 +1,9 @@
+vcpkg_download_distfile(PR_22011_PATH
+    URLS "https://patch-diff.githubusercontent.com/raw/openvinotoolkit/openvino/pull/22011.diff"
```

Needs `?full_index=1` for a stable hash, see #33305

---

Oh I knew the issue but couldn't find the PR. Appreciate the comment 👍 
```suggestion
    URLS "https://patch-diff.githubusercontent.com/raw/openvinotoolkit/openvino/pull/22011.diff?full_index=1"
```

---

I just copy-pasted the copied address for raw view. I think github.com is more portable one. let me change it